api/v1alpha1/etcdcluster_types.go

```diff
@@ -152,6 +152,13 @@ type PodSpec struct {
 	// see https://kubernetes.io/docs/concepts/containers/images/#referring-to-an-imagepullsecrets-on-a-pod
 	// +optional
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
+	// ServiceAccountName is the name of the ServiceAccount to use to run the etcd pods.
```

Comment:
If you are 'touching" podSpec with this MR, can I ask you to fix a bug, that image was left in the podSpec? We moved it to containers.